src/copilot/inspect/InspectionCopilot.ts

```diff
@@ -139,6 +196,19 @@ export default class InspectionCopilot extends Copilot {
         });
     }
 
+    private async doInspectRange(document: TextDocument, range: Range | Selection): Promise<Inspection[]> {
+        const adjustedRange = new Range(new Position(range.start.line, 0), new Position(range.end.line, document.lineAt(range.end.line).text.length));
+        const content: string = document.getText(adjustedRange);
+        const offset = range.start.line;
```

Comment:
offset -> startLine ?

offset is usually the absolute position in the document text, and offset can be converted to line & column position.

Reply:
renamed